sdk/openai/azure-ai-openai/CHANGELOG.md

```diff
@@ -4,6 +4,37 @@
 
 ### Features Added
 
+#### gpt-4-vision-preview
```

Comment:
Maybe we can copy what .NET released CHANGELOG content. 

See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/openai/Azure.AI.OpenAI/CHANGELOG.md#100-beta10-2023-12-06

Reply:
Sounds good! I took it instead from the PR description but this seems better!